skimage/measure/profile.py

```diff
@@ -63,20 +63,28 @@ def profile_line(image, src, dst, linewidth=1,
     array([1., 1., 1., 2., 2., 2.])
 
     For different reduce_func inputs:
+
     >>> profile_line(img, (1, 0), (1, 3), linewidth=3, reduce_func=np.mean)
     array([0.66666667, 0.66666667, 0.66666667, 1.33333333])
     >>> profile_line(img, (1, 0), (1, 3), linewidth=3, reduce_func=np.max)
     array([1, 1, 1, 2])
     >>> profile_line(img, (1, 0), (1, 3), linewidth=3, reduce_func=np.sum)
     array([2, 2, 2, 4])
 
-    The full array will be returned when reduce_func=None.
+    The full array will be returned when reduce_func=None and when reduce_func
+    acts on each pixel value individually.
```

Comment:
"full array" doesn't refer to `img` here but the internal array `pixels`, correct? In that case the wording might be a bit misleading. Perhaps replace "full array" with "the unreduced array". 

Oh and as a tip: if you surround parameter names with single backticks \` the name is emphasized when rendered by Sphinx (section "4. Parameters" in [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#sections)).

Reply:
> For the back ticks, you mean every time I refer to reduce_func in a comment, I should have it as \`reduce_func\` ?

Yes. It's suggested by [Sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=default_role#roles) and the recommended style by numpydoc. It's certainly not something enforced in scikit-image but I thought it worth mentioning. :)

Concerning the flipping of the last dimension: you are exactly right. Just forgot to consider functions that don't actually reduce but operate on each element individually.